src/de/mod.rs

```diff
@@ -2167,12 +2184,12 @@ where
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
 /// A structure that deserializes XML into Rust values.
-pub struct Deserializer<'de, R>
+pub struct Deserializer<'de, R, S: EntityResolver = DefaultEntityResolver>
```

Comment:
Let's use consistent naming
```suggestion
pub struct Deserializer<'de, R, E: EntityResolver = DefaultEntityResolver>
```